schemas/input/csl-data.json

```
@@ -4,425 +4,477 @@
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "authority": {
+          "type": "string"
+        },
+        "call-number": {
+          "type": "string"
+        },
+        "chapter-number": {
+          "type": ["string", "number"]
+        },
+        "citation-number": {
+          "type": ["string", "number"]
+        },
+        "citation-label": {
+          "type": "string"
+        },
+        "collection-number": {
+          "type": ["string", "number"]
+        },
+        "collection-title": {
+          "type": "string"
+        },
+        "container-title": {
+          "type": "string"
+        },
+        "container-title-short": {
+          "type": "string"
+        },
+        "dimensions": {
+          "type": "string"
+        },
+        "division": {
+          "type": "string"
+        },
+        "DOI": {
+          "type": "string"
+        },
+        "edition": {
+          "type": ["string", "number"]
+        },
+        "event": {
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "event-place": {
+          "description": "[Deprecated - Q: what TODO?]",
+          "type": "string"
+        },
+        "first-reference-note-number": {
+          "type": ["string", "number"]
+        },
+        "genre": {
+          "type": "string"
+        },
+        "ISAN": {
+          "type": "string"
+        },
+        "ISBN": {
+          "type": "string"
+        },
+        "ISCI": {
+          "type": "string"
+        },
+        "ISMN": {
+          "type": "string"
+        },
+        "ISRC": {
+          "type": "string"
+        },
+        "ISSN": {
+          "type": "string"
+        },
+        "ISWC": {
+          "type": "string"
+        },
+        "issue": {
+          "type": ["string", "number"]
+        },
+        "jurisdiction": {
+          "type": "string"
+        },
+        "keyword": {
+          "type": "string"
+        },
+        "locator": {
+          "type": ["string", "number"]
+        },
+        "medium": {
+          "type": "string"
+        },
+        "note": {
+          "type": "string"
+        },
+        "number": {
+          "type": ["string", "number"]
+        },
+        "number-of-pages": {
+          "type": ["string", "number"]
+        },
+        "number-of-volumes": {
+          "type": ["string", "number"]
+        },
+        "original": {
+          "title": "Original - Related Item",
+          "description": "Used to define metadata for an original publication.",
+          "$ref": "#/definitions/bibitem"
+        },
+        "original-publisher": {
+          "description": "[Deprecated - Use `publisher` on the `original` related item instead]",
+          "type": "string"
+        },
+        "original-publisher-place": {
+          "description": "[Deprecated - Use `publisher-place` on the `original` related item instead]",
```

```suggestion
          "description": "Can also be be entered as `publisher-place` on the `original` related item",
```

I think we could leave these `original-*` variables available for direct entry into item data.